src/constants/constants.go

```diff
@@ -17,6 +17,7 @@ const (
 	AvailableRateLimitKey            = "Available-Limit"
 	Password                         = "password"
 	OTP                              = "otp"
+	CreatedAt                        = "created_at"
```

Comment:
add this in constant group of 
// Column Name Constants
const (
already defined